### PR TITLE
Add python-gflags

### PR DESCRIPTION
**Pull request by jakirkham**

### CODE REVIEW

File: recipes/python-gflags/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - gflags
+
+about:
+  home: https://github.com/google/python-gflags
+  license: BSD
```
Comment (pelson) [inline recipes/python-gflags/meta.yaml]: Which one :wink:

**Comment by pelson** [comment]
Thanks @jakirkham. I've merged, as this was a nicely put together recipe. I did raise a question about the BSD version, which if you wanted to follow up on, can be done in the feedstock.

Thanks! :+1:

File: recipes/python-gflags/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipes/python-gflags/meta.yaml]: The linter should warn about that. I will open an issue and try to implement it.

Comment (jakirkham) [inline recipes/python-gflags/meta.yaml]: Looks like the standard [BSD license]( https://github.com/google/python-gflags/blob/master/COPYING ). I suppose I could say it is 3 clause. Though to be honest I often miss the little variations between the BSDs. Maybe one of you know?

**Comment by jakirkham** [comment]
I believe it is just a standard 3-Clause license. I'll add that in a PR. If either of you have more suggestions, let's meet over there will post a link in a moment.

**Comment by jakirkham** [comment]
Here is license clarification PR ( https://github.com/conda-forge/python-gflags-feedstock/pull/1 ).

**Comment by pelson** [comment]
Thanks for following up @jakirkham

**Comment by jakirkham** [comment]
Of course.

**Pull request merged**